tools/klee/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+add_executable(klee
+	main.cpp
+)
+
+set(LLVM_COMPONENTS
+	bitreader
+	bitwriter
+	engine
+	ipo
+	irreader
+	jit
+	linker
+	support
+)
+
+if (${LLVM_PACKAGE_VERSION} VERSION_LESS "3.5")
```

Comment:
Please encapsulate this call to `llvm_map_components_to_*()` in a function so we can reuse in other places.